quickstarts/redisenterprise/config.yml

```diff
@@ -0,0 +1,87 @@
+slug: Redis Enterprise  
+
+# Displayed in the UI (required)
+title: RedisEnterprise
+
+# Long-form description of the quickstart (required)
+description: |+
+  ## RedisEnterprise Quickstart
+
+  ### RedisEnterprise is an enterprise offering of Redis that provides:
+  - High availability: Up to 5 9's
+  - Support: 24/7 Enterprise support
+  - Geo-replication: Write data consistency across multiple data centers
+  - Tiered storage: Store data on devices outside of RAM for cost savings
+  - High performance: Serve up to 200M operations per second
+
+  This integration supports [Redis Enterprise Software](https://redis.com/redis-enterprise-software/overview/) installations.
+
+  ### With the Redis Enterprise quickstart you can:
+
+  - Monitor real-time performance and trends of you Redis Enterprise cluster
+  - Drill down into the performance of individual Redis databases
+  - Understand the Redis Active/Active metrics
+
+  ### Cluster level metrics
+
+  - License Status: Information on the Enterprise license status
+  - License Capacity: Capacity licensed
+  - Used Capacity: Capacity currently in use
+  - Cluster Capacity: Information on memory and compute resources available on the cluster
+  - Total Requests: Overall throughput of the cluster in total
+
+
+  ### Database level metrics
+
+  - Database Latency: Metrics pertaining to the latency of operations at the database level
+  - Database Throughput: Detailed information of the number of read/write operations per second
+  - Database Capacity: Memory usage details for capacity planning
+  - Network Usage: Information on the network usage per database
+  - Redis Data Retention: Expiration and Eviction statistics on object retention
+
+
+  ### Active/Active metrics
+
+  - Lag: Time between application on servers
+  - Pending:  Details of the number of writes pending as part of the sync
+  - Bandwidth:  Information on the amount of traffic between clusters 
+  
+  ### Requirements
+  To set up the integration, please see the [Full Documentation](https://github.com/redis-field-engineering/newrelic-redis-enterprise)
+  
+  ### Contact us
+  To get help please contact the [Redis Enterprise Field Engineering](mailto:redis.observability@redis.com?subject=NewRelic%20Integration%20Support)
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: This integration allows users to view Redis Enterprise analytics in the New Relic dashboard.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Redis
+  - Chris Mague
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - newrelic partner
+  - devops
+  - redisenterprise
+  - redis
+  - featured
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-redis-enterprise
```

Comment:
Looks like there is no file added to the install directory. You'll have to create an install plan so that users can install this quickstart. [Here](https://github.com/newrelic/newrelic-quickstarts/blob/main/install/third-party/fastly/install.yml) is an example you can reference.

Reply:
@maguec  looks like there is still no install file. The install file needs to be added into the install directory, once it's there I can verify if it works.

Reply:
@jcountsNR ah OK.  I had read that file incorrectly and now added install/redisenterprise/install.yml